### PR TITLE
Enabled background tile rendering. 

### PR DESCRIPTION
**Pull request by electronoob**
Example nametable, attribute, and 'first nes' text to screen. Added code to copy nametable and attributes. Code first clears nametable ram then copies bytes over.

### CODE REVIEW

File: data/background/scene.inc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+;
+; Sample nametable to produce a background scene.
+; Uses data/tiles/smb1_chr.bin
+;
+; Please see first_nes/docs/README.md for license and attribution.
+
+
+_NAMETABLE:
+  .BYTE $24,$24,$24,$24,$24,$24,$24,$24,$24,$24,$24,$24,$24,$24,$24,$24
+  .BYTE $24,$24,$24,$24,$24,$24,$24,$0f,$12,$1b,$1c,$1d,$24,$17,$0e,$1c ;first nes
```
Comment (gregkrsak) [inline data/background/scene.inc]: I can't recall-- I assume there is no underscore?

Also, maybe change the comment to clarify that it is actually referencing the glyphs for F-I-R-S-T and N-E-S, and not just making a vague comment/mention to the project name "first_nes".

File: first_nes.s [hunk 2]
```diff
@@ -73,6 +73,9 @@
 .SEGMENT "SPRITES"
 .INCLUDE "data/sprites/small_luigi.inc"
 
+; Background nametable and attribute data
+.INCLUDE "data/background/scene.inc"
+
```
Comment (gregkrsak) [inline first_nes.s]: Referencing https://github.com/gregkrsak/first_nes/blob/master/config/ines.cfg#L90 here...

Should we create a new segment identifier (vs. using SPRITES) for the background?

File: lib/isr/poweron_reset.s [hunk 3]
```diff
@@ -71,6 +71,8 @@
 
     jsr     LoadPaletteData
     jsr     LoadSpriteData                                
+    jsr     LoadBackgroundNametable
+    jsr     LoadBackgroundAttribute
```
Comment (gregkrsak) [inline lib/isr/poweron_reset.s]: 👍

File: first_nes.s (hunk 2 above)
Comment (electronoob) [reply first_nes.s]: youre right this should be done

File: data/background/scene.inc (hunk 1 above)
Comment (electronoob) [reply data/background/scene.inc]: no underscore in the original that I could see.
perhaps a new tileset is in order?

and youre right, that documentation was poor at best.

**Comment by electronoob** [comment]
yeah im cool with whatever you decide. happy to help.

**Comment by gregkrsak** [comment]
I haven't forgotten about this. Forgive me for being distracted+busy.

**Comment by electronoob** [comment]
no problem at all, life tends to get in the way.

**Pull request merged**